packages/terra-clinical-alert/tests/jest/Alert.test.jsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+/* eslint-disable import/no-extraneous-dependencies */
+import { IntlProvider } from 'react-intl';
+
```

Comment:
Its seems like you added tests to ensure terra-alert alerts are rendered correctly, do we want to double test these? For example, you have the test `'Alert of type success with an action button text content'`. If we do, we are missing some of the alert types